lib/hanami.rb

```diff
@@ -113,8 +113,7 @@ def self.shutdown
   def self.bundled?(gem_name)
     @_mutex.synchronize do
       @_bundled[gem_name] ||= begin
-        gem(gem_name)
-        true
+        Bundler.gem(gem_name)
```

Comment:
@solnic I was fine for us to merge this PR wholesale to get the main branch working again, but I'm interested in learning why you had to make this change?

Reply:
@timriley looks like it was not needed. I changed it back in main.